usr/share/rear/output/PXE/default/810_create_pxelinux_cfg.sh

```diff
@@ -102,7 +102,7 @@ popd >/dev/null
 
 if test "$PXE_CONFIG_URL" ; then
     LogPrint "Created PXELINUX config '$pxe_config_file' and symlinks for $PXE_CREATE_LINKS adresses in $PXE_CONFIG_URL"
-    umount_url $PXE_TFTP_UPLOAD_URL $BUILD_DIR/tftpbootfs
+    umount_url "$PXE_TFTP_UPLOAD_URL" $BUILD_DIR/tftpbootfs
```

Comment:
`$BUILD_DIR/tftpbootfs` should be quoted as well for consistency with other changes